### ISSUE TITLE
First Public {WG|IG} Note status is missing

### ISSUE THREAD

**Issue by svgeesus** [tabatkins/bikeshed#1836]
[Documented statuses](https://tabatkins.github.io/bikeshed/#metadata) are:

```
    ED: "Editor’s Draft"

    WD: "W3C Working Draft"

    FPWD: "W3C First Public Working Draft"

    LCWD: "W3C Last Call Working Draft"

    CR: "W3C Candidate Recommendation"

    PR: "W3C Proposed Recommendation"

    REC: "W3C Recommendation"

    PER: "W3C Proposed Edited Recommendation"

    WG-NOTE: "W3C Working Group Note"

    IG-NOTE: "W3C Interest Group Note"

    NOTE: "W3C Note" (unclear what this is used for)

    MO: "W3C Member-only Draft"

    UD: "Unofficial Proposal Draft"

    CG-DRAFT: "Draft Community Group Report"

    CG-FINAL: "Final Community Group Report"
```

Missing are FP-WG-NOTE and FP-IG-NOTE. This means the boilerplate is incorrect, and the pubrules checker complains that "Previous Version" is missing. While that can be corrected manually (and First Public notes have to be manually published, currently) it would be better if Bikeshed generated the correct stuff.

**Comment by svgeesus** [comment]
Also while I am here, please make any draft with `status=MO` emit a warning "are you sure you want to do this?"

**Comment by svgeesus** [comment]
An example of [pubrules failures caused by the lack of FP-WG-NOTE](https://github.com/w3c/csswg-drafts/issues/4715#issuecomment-746828760)

**Comment by jyasskin** [comment]
I think this was fixed by #2019.
